src/lottie/lottieitem.h

```diff
@@ -348,6 +348,8 @@ class LOTContentGroupItem: public LOTContentItem
    LOTGroupData                                  *mData{nullptr};
    std::vector<LOTContentItem*>   mContents;
    VMatrix                                        mMatrix;
+private:
+   LOTProxyModel<LOTGroupData> mModel{nullptr};
```

Comment:
How this non pointer instance can be nullptr? Is it correct?

Reply:
As far as i know. this is correct.
please see lottieproxymodel.h 244 line

Reply:
lottieproxymodel.h 244 line => see what?
244         return LottieColor(col.r(), col.g(), col.b());

Reply:
I update code. Thank you for your review.